runtime/nvqir/cutensornet/simulator_cutensornet.h

```diff
@@ -52,7 +52,7 @@ class SimulatorTensorNetBase : public nvqir::CircuitSimulatorBase<double> {
   virtual cudaq::ExecutionResult observe(const cudaq::spin_op &op) override;
 
   /// @brief Add qubits to the underlying quantum state
-  virtual void addQubitsToState(std::size_t count) override;
```

Comment:
I had a build error from this being marked as override. Not sure if anyone else ran into this